Sources/App/Commands/Analyzer.swift

```diff
@@ -97,6 +97,18 @@ func analyze(application: Application, packages: [Package]) -> EventLoopFuture<V
                 .map { getManifests(logger: application.logger, versions: $0) }
                 .flatMap { updateVersionsAndProducts(on: tx, packages: $0) }
                 .flatMap { updateLatestVersions(on: tx, packages: $0) }
+                .flatMap { input in
+                    let ops: [EventLoopFuture<Void>] = input.compactMap {
+                        guard let package = try? $0.get() else {
+                            return nil
+                        }
```

Comment:
This is a bit problematic, because it'll strip off and discard errors. I look into rebasing this onto my `onVersionChange` thingie and extend it to pass in the package or something.